lib/Page.js

```diff
@@ -800,11 +800,27 @@ class Page extends EventEmitter {
   }
 
   /**
-   * @param {?string} mediaType
+   * @param {?string} type
    */
-  async emulateMedia(mediaType) {
```

Comment:
Shouldn't we retain `emulateMedia` and mark it deprecated?

Reply:
That's part of what I'd like to discuss here. I think we should definitely retain it for v1. Then again, the upcoming v2 gives us a potential "clean break" opportunity.

Reply:
Sounds good. I like the separation of the two APIs reflected explicitly in puppeteer. :+1:

Reply:
Is there a tracking issue documenting that we should remove `emulateMedia` for 2.0?